apps/store-app/src/app/entities/requisition.entity.ts

```diff
@@ -0,0 +1,27 @@
+import 'reflect-metadata';
```

Comment:
I have already commented something similar in other PR but angular is getting rid of this polyfils.  Should not we do the same? 

https://angular.io/guide/deprecations#reflect-metadata